NodeFirefoxDebug/entry_point.sh

```diff
@@ -29,22 +29,16 @@ if [ ! -z "$SE_OPTS" ]; then
   echo "appending selenium options: ${SE_OPTS}"
 fi
 
-# TODO: Look into http://www.seleniumhq.org/docs/05_selenium_rc.jsp#browser-side-logs
-
 SERVERNUM=$(get_server_num)
 
 rm -f /tmp/.X*lock
 
-env | cut -f 1 -d "=" | sort > asroot
-  sudo -E -u seluser -i env | cut -f 1 -d "=" | sort > asseluser
-  sudo -E -i -u seluser \
-  "$(for E in $(grep -vxFf asseluser asroot); do echo $E=$(eval echo \$$E); done)" \
-  DISPLAY=$DISPLAY \
```

Comment:
This was a workaround to export the ENV vars present in the `root` user and not present in the `seluser`. Since all ENV vars are now exported with the `seluser`, this is not needed anymore.